volume/drivers/aws/aws.go

```diff
@@ -203,11 +205,13 @@ func (d *Driver) Create(
 	if *volType != opsworks.VolumeTypeGp2 {
 		ec2Vol.Iops = iops
 	}
-	vol, err := d.ops.Create(ec2Vol, locator.VolumeLabels)
+	resp, err := d.ops.Create(ec2Vol, locator.VolumeLabels)
 	if err != nil {
 		dlog.Warnf("Failed in CreateVolumeRequest :%v", err)
 		return "", err
 	}
+
+	vol := resp.(*ec2.Volume)
```

Comment:
Doesn't hurt to add the type cast check here as well even if ops.Create is going to return an ec2.Volume